workshop/SCHEDULE.md

```diff
@@ -56,32 +56,34 @@ Minneapolis, MN 55414 in Room 105.
-| 10:00 AM    | RNA-Seq import with tximeta |
+| 9:00 AM     | [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
+| 10:00 AM    | [RNA-Seq import with tximeta](../completed-notebooks/RNA-seq/02-gastric_cancer_tximeta.nb.html) |
 | 10:45 AM    | Coffee break |
-| 11:00 AM    | RNA-Seq exploratory analysis |
-|             | Exercise: Neuroblastoma cell line tximeta |
-|             | Exercise: Exploratory data analysis |
+| 11:00 AM    | [RNA-Seq exploratory analysis](../completed-notebooks/RNA-seq/03-gastric_cancer_exploratory.nb.html) |
+|             | **TODO this isn't an exercise anymore?** [Exercise: Neuroblastoma cell line tximeta](../completed-notebooks/RNA-seq/04-nb_cell_line_tximeta.nb.html) |
+|             | [Exercise: Exploratory data analysis](https://github.com/AlexsLemonade/training-modules/blob/2024-august/RNA-seq/exercise_01-exploratory_data_analysis.Rmd) |
 | 12:00 PM    | Lunch break |
 | 1:00 PM     | Set up Git on RStudio Server <br> [Git setup slides (PDF)](../slides/2024-08-21_git-server-setup.pdf) |
-| 1:30 PM     | Differential expression analysis |
+| 1:30 PM     | [Differential expression analysis](../completed-notebooks/RNA-seq/05-nb_cell_line_DESeq2.nb.html) |
 | 2:45 PM     | Coffee break |
-| 3:00 PM     | Heatmaps and clustering |
+| 3:00 PM     | [Heatmaps and clustering](../completed-notebooks/RNA-seq/06-openpbta_heatmap.nb.html) |
 | 4:00 PM     | Questions and consultation time |
-|             | Exercise: Bulk RNA-seq |
+|             | [Exercise: Bulk RNA-seq](https://github.com/AlexsLemonade/training-modules/blob/2024-august/RNA-seq/exercise_02-bulk_rnaseq.Rmd) |
 | 5:00 PM     | Adjourn |
 
 | Time        | Topic                                          |
 |-------------|------------------------------------------------|
 | **Day 4**   | **2024-08-22 <br> Bulk RNA-Seq: Pathway Analysis <br> Participant Presentations** |
 | 8:30 AM     | Breakfast and coffee available |
 | 9:00 AM     | Questions and review |
-| 9:30 AM     | Introduction to pathway analysis <br> [Introduction to Pathway Analysis slides (PDF)](../slides/2024-08-22_intro-to-pathway-analysis.pdf) |
-| 10:00 AM    | Pathway analysis: Over-representation analysis |
+| 9:30 AM     | Introduction to pathway analysis <br> [Introduction to pathway analysis slides (PDF)](../slides/2024-08-22_intro-to-pathway-analysis.pdf) |
+| 10:00 AM    | [Pathway analysis: Over-representation analysis](../completed-notebooks/pathway-analysis/01-overrepresentation_analysis.nb.html) |
 | 11:00 AM    | Coffee break |
-| 11:15 AM    | Pathway analysis: <br> Gene Set Enrichment Analysis <br> Gene Set Variation Analysis |
+| 11:15 AM    | Pathway analysis: <br> [Gene set enrichment analysis](../completed-notebooks/pathway-analysis/02-gene_set_enrichment_analysis.nb.html) <br> [Gene Set Variation Analysis](../completed-notebooks/pathway-analysis/03-gene_set_variation_analysis.nb.html) |
```

Comment:
This was capitalized for acronym/method name reasons, but I am kind of agnositc. I'd probably take off the `Pathway analysis:` prefix on these though (and ORA). 


```suggestion
| 11:15 AM    |  [Gene Set Enrichment Analysis](../completed-notebooks/pathway-analysis/02-gene_set_enrichment_analysis.nb.html) <br> [Gene Set Variation Analysis](../completed-notebooks/pathway-analysis/03-gene_set_variation_analysis.nb.html) |
```